### PR TITLE
Components for header, footer

### PR DESCRIPTION
**Pull request by Tomerozfresko**
1. Created component for footer, header
2. Moved raw data to app/DUMMY_DATA.tsx

### CODE REVIEW

File: app/routes/index.tsx [hunk 1]
```diff
@@ -1,526 +1,47 @@
...
+//   useEffect(() => {});
+//   return (
+//     <Layout>
+//     </Layout>
+//   );
+// }
+
+import { perks, collections, trendingProducts } from "~/DUMMY_DATA";
+
+export default function Main() {
```
Comment (nir2002) [inline app/routes/index.tsx]: call it Home please

File: app/DUMMY_DATA.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
+export const perks = [
```
Comment (nir2002) [inline app/DUMMY_DATA.tsx]: put this file under "mocks" folder

File: app/components/Layout.tsx [hunk 3]
```diff
@@ -0,0 +1,586 @@
...
+  ],
+};
+
+function Header() {
+  const [open, setOpen] = useState(false);
+
+  return (
+    <>
+      {/* Mobile menu */}
+      <Transition.Root show={open} as={Fragment}>
```
Comment (nir2002) [inline app/components/Layout.tsx:227]: Can we split all this code into smaller components? for example `Menu` which will hold line 115 till about 365? or something like that, not sure

File: app/models/collection.client.ts [hunk 4]
```diff
@@ -1,45 +1,76 @@
+// import { runQuery } from "./utils";
```
Comment (nir2002) [inline app/models/collection.client.ts]: why is my code commented out?

File: app/routes/index.tsx [hunk 5]
```diff
@@ -1,526 +1,47 @@
...
-                        </span>
-                        <span className="sr-only">items in cart, view bag</span>
-                      </a>
-                    </div>
-                  </div>
-                </div>
-              </div>
+    <main>
+      {/* Hero section */}
+      <div className="relative">
```
Comment (nir2002) [inline app/routes/index.tsx]: split this into components

**Review by nir2002 — CHANGES_REQUESTED** [review]

File: app/routes/index.tsx (hunk 1 above)
Comment (Tomerozfresko) [reply app/routes/index.tsx]: Component is Home

File: app/components/Layout.tsx (hunk 3 above)
Comment (Tomerozfresko) [reply app/components/Layout.tsx:227]: New components in Header: "NavBar", "MobileMenu", "Cart". 
They are using the same "open" state. (Except the Cart)
Is it fine to define components under the default function?

File: app/models/collection.client.ts (hunk 4 above)
Comment (Tomerozfresko) [reply app/models/collection.client.ts]: Mistakenly committed, test code. 
Reverted.

File: app/routes/index.tsx (hunk 5 above)
Comment (Tomerozfresko) [reply app/routes/index.tsx]: Split to "Collections" & "Trending", as per the natural layout of this page.

**Review by nir2002 — APPROVED** [review]

**Review by nir2002 — APPROVED** [review]

**Pull request merged**